package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "talkly",
+  "scripts": {
+    "postinstall": "./node_modules/.bin/bower install && ./node_modules/.bin/gulp",
+    "start": "./node_modules/.bin/gulp watch"
+  },
+  "dependencies": {
+    "bower": "^1.7.9",
+    "gulp": "^3.9.0",
+    "gulp-concat": "^2.6.0",
+    "gulp-minify-css": "^1.2.1",
```

Comment:
@DavidBadura this gulp plugin is deprecated! But why don't you use the `outputStyle: compressed`-Option https://github.com/sass/node-sass#outputstyle of gulp-sass/node-sass which is written in c/c++?